ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -278,6 +278,22 @@ impl<P: Parameters> Default for GroupAffine<P> {
     }
 }
 
+impl<P: Parameters> core::iter::Sum<Self> for GroupAffine<P> {
+    fn sum<I: Iterator<Item = Self>>(iter: I) -> Self {
+        iter.map(|x| x.into_projective())
+            .sum::<GroupProjective<P>>()
+            .into()
+    }
+}
+
+impl<'a, P: Parameters> core::iter::Sum<&'a Self> for GroupAffine<P> {
+    fn sum<I: Iterator<Item = &'a Self>>(iter: I) -> Self {
+        iter.map(|x| x.into_projective())
+            .sum::<GroupProjective<P>>()
```

Comment:
It would be faster to use mixed addition, no?

Reply:
Good point. Let me double-check. Yes, it is very likely that the conversions from affine to projective may be unnecessary as well.

Reply:
(though the latter is cheap)

Reply:
Done in #291